content/blog/2022/11/ui-1.0.0-release.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Kubewarden UI 1.0.0 is released!
+authors:
+- Jordon Leach 
+date: 2022-11-01 # TODO: Update date before publishing
+---
+
+We are excited to announce that the Kubewarden UI 1.0.0 has been released!
+
+The UI is an Extension for [Rancher Manager](https://github.com/rancher/rancher/), now you will be able to enable Kubewarden policies for your Kubernetes clusters with a streamlined user experience.
+
+Let's take a look at what is achievable from the UI as a Cluster Admin.
+
+## Manage Policy Servers
+
+All of your Policy Servers are listed in an overview page with readily available details of each resource including the status of related policies.
+
+{{<figure src="/images/ui/policyserver-list.png" alt="policy servers list view">}}
```

Comment:
Wild request: How about doing 1 or 2 gifs? :D moving shiny things capture attention.
If not, I wonder if we could make all the screenshots the same size, so they flow better in the post, or a `center` for them.

I personally wouldn't mind showing the rest of Rancher Explorer for context, but wonder what you all think.

Reply:
I added a couple of brief videos for the installation and creating policies, along with resizing the images to all be the same width. Let me know what you think!

Reply:
looks good here! resolving